packages/gibsonify_api/lib/src/models/gibsons_form.dart

```diff
@@ -10,6 +10,7 @@ import 'food_item.dart';
 class GibsonsForm extends Equatable {
   GibsonsForm(
       {String? id,
+      this.employeeNumber,
```

Comment:
This looks nice, I was thinking maybe in the future we might add some restrictions into Collection if someone with different employee id tries to edit a collection or something like that. Or to have at least a confirmation.

Reply:
Yeah, employee number is not currently visible within the Recipe module either but would be worth adding it as a disabled textfield for viewing purposes